manuscript/chapter3-contributor-mode.md

```diff
@@ -39,10 +39,29 @@ For Italian people usually those rules are enough to understand how to behave di
 
 ![Body languages difference all around the world](images/3/body-language.jpg)
 
+Basically Open Source is a group work with different people so is important to have some basics rules to live and moving on and avoid problems during the discussions that can degenerate on flame war. A reading about [work with people that you don't like can be easily found here](https://hbr.org/2018/12/how-to-collaborate-with-people-you-dont-like).
+
+### Understand if a project is alive
+
+One of the common issues in Open source is to understand if a project is alive, abandoned or dead.
+
+When is alive is easy to see, the tickets are getting replies in reasonable time (like 1 month at maximum), the pull requests are getting reviews, the social channels keep pushing updates when there are (and not forgetting to do so) and so on.  
+
+Abandoned is more complicated because maybe the owner or maintainer is not replying on the tickets or pull requests but is active on social networks. So basically seems that is not caring at all of the project but he wrote anything about it in the project page itself. Usually this means that a fork is possible without so many troubles but probably will be difficult to get the ownerships of the website domain, social accounts etc from the original creator.  
```

Comment:
You might want to mention troubles with naming policies or ownership transfer on registries (like npm, pypi, PEAR etc) for forked projects.

I found Python difficult, for example. Since they state in [PEP 541](https://www.python.org/dev/peps/pep-0541/#implementation), that the former owner is not reachable before you can claim it.
But what if you can reach him, but he don't want to continue?

I'd not cover it in this section, though.